webrtc/_compositions/go2go.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+  name = "go2go {{ .Env.GitReference }}"
+
+[global]
+  plan = "libp2p/webrtc"
+  case = "webrtc"
+  builder = "docker:go"
+  runner = "local:docker"
+  total_instances = 2
+
+  [global.build_config]
+    enable_go_build_cache  = false # see https://github.com/testground/testground/issues/1361
+    go_proxy_mode          = "remote"
+    go_proxy_url           = "https://proxy.golang.org"
+
+[[groups]]
+  id = "server"
+  instances = { count = 1 }
+  builder = "docker:go" # override builder here
```

Comment:
no need to define, right? As it's already `docker:go` by default

Reply:
Yes you're probably right about that. "by default" meaning it gets picked up from the plan, though, right?

Reply:
It's because you define it as the default plan in `[global]`

Reply:
Yeah I think what happened here is the original mixed & matched different implementations. So it couldn't be just the global setting. 
But since at the moment I'm _only_ dealing with go, I could drop this.